### DISCUSSION TITLE
Query Federation - Join data from multiple data-sources

### DISCUSSION THREAD

**Discussion by MichalisDBA** [9tigerio/db2rest#927] [Ideas]
It would be great to have the ability to query and join data across multiple data-sources. 

An example of that implementation is used by https://trino.io/

An example query would be"

```
SELECT * FROM mysql.db.table -- MySQL Server
LEFT JOIN mssql.db.table -- Microsoft SQL Server
ON mysql.db.table.id = mssql.db.table.id.id
```

```
SELECT * FROM mysql_srv1.db.table -- MySQL Server 1
LEFT JOIN mysql_srv2.db.table -- MySQL Server 2
ON mysql_srv1.db.table.id = mysql_srv2.db.table.id.id
```

Except tables, support also VIEWS.

**Comment by thadguidry** [comment]
I think this is an enhancement that we could support really well in our Enterprise edition.  Would you be able to sponsor the work?
